movie-app/src/scss/components/body/_card.scss

```diff
@@ -1,105 +1,108 @@
 .card {
-    background-color: #ccc;
-    background-size: cover;
-    color: rgb(226, 226, 226);
-    height: 450px;
-    display: flex;
-    flex-direction: column;
-    justify-content: flex-end;
-    align-items: flex-start;
-    position: relative;
-    overflow: hidden;
+  background-color: #ccc;
+  background-size: cover;
+  color: rgb(226, 226, 226);
+  height: 450px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: flex-start;
+  position: relative;
+  overflow: hidden;
+  min-height: 450px;
+  min-width: 300px;
+  max-height: 450px;
+  max-width: 300px;
```

Comment:
Let me know how this works out!  Sometimes CSS can get a bit finnicky even with min-max constraints.

Reply:
I was able to convert to em and it works a lot better.